server/app/routes/games.router.js

```diff
@@ -37,9 +36,19 @@ router.get('/:id', function(req, res, next){
 })
 
 router.post('/', function(req, res, next){
+  let invitedPlayers = req.body.players.invited.forEach(u=>u.id);
   Game.create(req.body.game)
-  .then(game=>game.setPlayers(req.body.players))
-  .then(game=> res.send(game))
+  .tap(game=>game.setInvitees(invitedPlayers))
+  .tap(game=>game.setPlayers(req.body.players.accepted))
+  .then(game=>game.setCommissioner(req.body.commissioner))
+  .tap(game=>{
+    let taskProms = req.body.tasks.map(taskObj=>Task.create(taskObj));
+    return Promise.all(taskProms).map(function(task, index) {
```

Comment:
I believe the `.map(...)` code could actually be placed on the outer promise chain (i.e. between lines 49 and 50 below).